### PR TITLE
Update shell scripts to use module path.

### PR DESCRIPTION
**Pull request by bravehorsie**
Added dependency to javax.annotation (fix for jdk without java.xml.bind module).

Signed-off-by: Roman Grigoriadi <roman.grigoriadi@oracle.com>

### CODE REVIEW

File: jaxb-ri/bundles/ri/src/main/resources/bin/schemagen.sh [hunk 1]
```diff
@@ -72,37 +72,71 @@ then
...
-    LOCALCLASSPATH=`cygpath -w -p ${LOCALCLASSPATH}`
+    LOCALPATH=`cygpath -w -p ${LOCALPATH}`
 }
 
 if [ `expr \`uname\` : 'CYGWIN'` -eq 6 ]
 then
     JAXB_HOME="`cygpath -w "$JAXB_HOME"`"
 fi
 
+VERSION_STRING=`${JAVA} -version 2>&1 | head -n 1 | sed -E 's/.+version "(.+)"/\1/'`
```
Comment (lukasj) [inline jaxb-ri/bundles/ri/src/main/resources/bin/schemagen.sh]: Any special reason why `java -fullversion` is not used?

File: jaxb-ri/xjc/pom.xml [hunk 2]
```diff
@@ -345,6 +345,22 @@
...
                         </goals>
                     </execution>
+                    <execution>
+                        <id>update-mr-jar</id>
+                        <phase>verify</phase>
+                        <configuration>
+                            <tasks>
+                                <jar destfile="${project.build.directory}/jaxb-xjc-${project.version}.jar" update="true">
+                                    <manifest>
+                                        <attribute name="Multi-Release" value="true"/>
```
Comment (lukasj) [inline jaxb-ri/xjc/pom.xml]: Shouldn't this be added by `maven-jar-plugin`?

File: jaxb-ri/bundles/ri/src/main/resources/bin/schemagen.sh (hunk 1 above)
Comment (bravehorsie) [reply jaxb-ri/bundles/ri/src/main/resources/bin/schemagen.sh]: We are interested only in the "major" version part so -version is sufficient, but I can update to fullversion and remove "head -n 1" than from parsing.

File: jaxb-ri/xjc/pom.xml (hunk 2 above)
Comment (bravehorsie) [reply jaxb-ri/xjc/pom.xml]: Good point, thanks @lukasj

**Pull request merged**